src/auth-service/bin/www

```diff
@@ -30,16 +51,74 @@ try {
 }
 
 
-
 /****
  * pass the app instance to the middleware configuration
  */
 try {
     middlewareConfig(app);
 } catch (e) {
-    console.log("middleware error", e.message);
+    logElement("middleware error", e.message)
 }
 
+
+const consumer = kafka.consumer({ groupId: constants.UNIQUE_CONSUMER_GROUP });
+
+// Function to start consuming messages
+const run = async() => {
+    await consumer.connect();
+    await consumer.subscribe({ topic: constants.NEW_MOBILE_APP_USER_TOPIC, fromBeginning: true });
+    await consumer.run({
+        eachMessage: async({ topic, partition, message }) => {
+            logger.info(
+                `KAFKA: successfully received the new User --- ${JSON.stringify({
+                    value: message.value.toString(),
+                })}`
+            );
+            logObject("message.value.toString()", message.value.toString())
+
+            /**
+             * We might need to "clean" the JSON from here
+             */
+
+            const repairedJSONString = jsonrepair(message.value.toString())
```

Comment:
I added the json repair as I was getting some weird json outputs @Baalmart

Reply:
Alright, thanks @BenjaminSsempala